ACE/ace/config-android.h

```diff
@@ -123,6 +123,10 @@
 #  endif
 #endif
 
+#if __ANDROID_API__ < 24
```

Comment:
NDK r12, the minimum revision that I consider ACE to support, has API 24, but it doesn't have `if_nameindex`, so this also depends on the NDK revision. I checked the NDKs after r12 and it appears in r14:

```suggestion
#if ACE_ANDROID_NDK_LESS_THAN(14, 0) || __ANDROID_API__ < 24
```

I would have just let my matrix catch this but something's wrong with it at the moment.